python/cudf/cudf/core/dataframe.py

```diff
@@ -668,22 +668,32 @@ def get_renderable_dataframe(self):
         ncols = (
             pd.options.display.max_columns
             if pd.options.display.max_columns
-            else 15
+            else pd.options.display.width / 2
         )
-        if len(self) <= nrows or len(self.columns) <= ncols:
+        if (
+            len(self) <= nrows
+            and len(self.columns) <= ncols
+            or isinstance(self.columns, cudf.MultiIndex)
```

Comment:
Why do we need to handle MultiIndex separately? We should be able to slice it similarly to a DataFrame, no?

Reply:
I'm handling MultiIndex separately here because updating the logic for slicing it depends on the upcoming changes that @shwina is working on that will drop any present or future support for multi-named columns. I feel like once that is in place it should be straight forward to support slicing of MultiColumn `DataFrames` as well.

Reply:
This is fixed in the latest.